Dockerfile

```diff
@@ -7,13 +7,23 @@ RUN npm run generate
 
 ### STAGE 1: Build server ###
 FROM node:16-alpine
-RUN apk update && apk add --no-cache --update ffmpeg
 ENV NODE_ENV=production
+RUN apk update && \
+    apk add --no-cache --update \
+    curl \
+    tzdata \
```

Comment:
What does timezone do for us here? Is that for the healthcheck?

Reply:
It allows me to change the timezone inside the container using an environment var. From what I've heard the server doesnt respect the timezone yet but it can be usefull for running other commands. I also wanted to look into seeing what it would take to use the local timezone in the logs.

Reply:
I can remove it if you want.

Reply:
I'm not sure how that works. If we wanted to add the timezone as an environment var wouldn't we need to set env using the `ENV` command?

Reply:
The tzdata package automatically picks up the `TZ` env and uses it to set the timezone inside the container.